ethereum/contracts/Ownable.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity 0.5.10;
+
+contract Ownable {
+  address public owner;
```

Comment:
This is very important. Would be a shame if a child contract (Pausable or Spitter) accidentally stomped on it because the developer didn't know `owner` is reserved (or wants to do some underhanded trickery). 

You can prevent that with `private`. 

It's usually a good idea to add a way to discover the value of private state variables, so `function getOwner() ...`